lux/action/column_group.py

```diff
@@ -26,6 +26,7 @@ def column_group(ldf):
     recommendation = {
         "action": "Column Groups",
         "description": "Shows charts of possible visualizations with respect to the column-wise index.",
+        "long_description": "Shows charts of possible visualizations with respect to the column-wise index.",
```

Comment:
Might want to expand on this description with an image or longer description and link to the description on [this page](https://lux-api.readthedocs.io/en/latest/source/advanced/indexgroup.html).

Reply:
@dorisjlee Added some descriptions (I'm not a fan of the images there, but we can chat about that more) as well as a link.